R/estimateDiversity.R

```diff
@@ -33,12 +33,23 @@
 #'   
 #' @param tree_name a single \code{character} value for specifying which
 #'   rowTree will be used to calculate faith index. 
-#'   (By default: \code{tree_name = "phylo"})
+#'   (By default: \code{tree_name = "phylo"})(Please use \code{tree.name} instead. At some point \code{tree_name}
+#'   will be disabled.)
```

Comment:
You could keep this as simple as possible. "Alias for tree.name. Please use tree.name instead...."